### PR TITLE
Less code

### PR DESCRIPTION
**Pull request by n10v**
Less code in `Certifi.where` method

### CODE REVIEW

File: lib/certifi.rb [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 module Certifi
   def self.where
-    File.expand_path(File.join(File.dirname(__FILE__), "certifi/vendor/cacert.pem"))
+    File.expand_path("certifi/vendor/cacert.pem", __dir__)
```
Comment (Lukasa) [inline lib/certifi.rb:5]: Uh...are the arguments to expand_path not backward here?

Comment (Lukasa) [inline lib/certifi.rb:5]: Aha, no, you're right, they aren't.

**Comment by Lukasa** [comment]
Hmm, **dir** only exists in Ruby 2.X. While strictly the older versions are not supported by Ruby core, I don't think we need to break certifi on them without good reason. I don't think this change is worth the effort, sadly. =( Sorry!

**Comment by n10v** [comment]
Okay. No problem

**Pull request closed without merging**